src/Tribe/Shortcodes/User_Event_Confirmation_List.php

```diff
@@ -65,10 +65,21 @@ public function generate( $params ) {
 	 * @param $params
 	 */
 	protected function set_params( $params ) {
-		$this->params = shortcode_atts( array(
-			'limit' => -1,
+		/**
+		 * Allow filtering of the default limit for the [tribe-user-event-confirmations] shortcode.
+		 * @since TBD
```

Comment:
:no_entry_sign: There must be exactly one blank line before the tags in a doc comment

<sub>hash: 68d4c40465b45164ef6718ff2a23b28c</sub>